Improve React Native support (#379)
coveo/coveo.analytics.js: README.md

```

```js
import {CoveoAnalyticsClient} from 'coveo.analytics';

// Create an API client
const client = new CoveoAnalyticsClient({token: 'YOUR_API_KEY'});
// Send your event
client.sendCustomEvent({
    eventType: 'dog',
    eventValue: 'Hello! Yes! This is Dog!',
    language: 'en',
});
```

## Using React Native

> Since React Native is not running inside a browser, it cannot use cookies or the local/session storage that modern browsers provide.
> It is necessary to provide your own Storage implementation. Thankfully there exist multiple packages to store data:

Reviewer: ```suggestion
Since React Native does not run inside a browser, it cannot use cookies or the local/session storage that modern browsers provide.
You must provide your own Storage implementation. Thankfully, there exist multiple packages to store data:
```
